Day-I-Social-Card/social-card/src/components/FooterComponents/FooterIcons.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Footer.css';
+
+const FooterIcons = () => {
+    return (
+        <div className='icons'>
+            <img className='msg' src="https://static.thenounproject.com/png/3766-200.png" alt="message"></img>
```

Comment:
this is okay for now, but you are taking people's property without permission, so be aware you'll need to find a legal way to get icons in your project in the real world